.github/workflows/test.yml

```diff
@@ -25,5 +25,11 @@ jobs:
       - name: Install dependencies
         run: yarn
 
-      - name: Run the tests
+      - name: Run the unit tests
         run: yarn test
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          start: yarn start
```

Comment:
@tanvirchahal I expected `yarn e2e` command here. Why we have `yarn start` which is for starting the server?

Reply:
@anuveyatsu i tried running tests without `yarn start` and i get the following error 

`Cypress could not verify that this server is running:
 > http://localhost:3000/`
 
 Full error log: https://github.com/datopian/data-explorer-graphql/runs/2677912757?check_suite_focus=true
 
 I think it's important to run the server before cypress can get started. 
 
 I got the yarn start code from official documentation for cypress: https://docs.cypress.io/guides/continuous-integration/github-actions#Basic-Setup

Reply:
OK got it now 👍